### DISCUSSION TITLE
是否在游戏下载界面加入关于MINECRAFT衍生游戏的下载

### DISCUSSION THREAD

**Discussion by xudisen** [Meloong-Git/PCL#739] [提问] [answered]
我还是蛮希望加入这个的，你知道的，官方的东西不一定是好货..........当然下载后能在启动器中运行那就是最好的事了awa

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #81 ，PCL2 不会提供 BE、MCD 等游戏版本或衍生作品的下载、启动服务
PCL2 是 Minecraft Java 版的启动器，这个功能有些超出 PCL2 的范畴了
（您想，连基岩版 PCL2 都不会支持提供下载与启动服务，更别提什么衍生游戏了qwq……
